src/main/java/com/kodilla/ecommercee/domain/OrderEntity.java

```diff
@@ -22,5 +23,24 @@
     @GeneratedValue
     private long id;
 
+    @GeneratedValue
+    @NotNull
+    private int orderNumber;
+
     private String productGroup;
+    private int price;
+
+    @OneToMany(
+            targetEntity = Product.class,
+            cascade = CascadeType.ALL,
+            fetch = FetchType.LAZY
+    )
+    @JoinColumn(name = "PRODUCT_ID")
```

Comment:
Najlepiej trzymać się jednego sposobu formatowania nazwy kolumny, małe albo duże litery. Sam osobiście używam małych, myślę, że trzeba to uzgodnić na rozmowie :)

Reply:
zgadzam się z @pastorek89

Reply:
Zmieniłem na małe litery.